sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
+  `DESC` TEXT,
+  [DESC] TEXT AS java.util.List
```

Comment:
Follow up: can we determine this is a raw parameterized type and fail the compiler?